app/src/main/java/org/simple/clinic/home/patients/PatientsEvent.kt

```diff
@@ -17,4 +18,8 @@ data class UserDetailsLoaded(val user: User): PatientsEvent()
 
 object ActivityResumed: PatientsEvent()
 
-data class DismissedApprovalStatusLoaded(val hasBeenDismissed: Boolean): PatientsEvent()
+data class RequiredInfoForShowingApprovedStatus(
```

Comment:
Rename this to an event name like `DataForShowingApprovedStatusLoaded`

Reply:
Done.